### ISSUE TITLE
Files list indicators labels

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/web#3673]
Instead of labelling the whole indicators row with `State`, there should be a label for each indicator. This requires extending the extension point of indicators.

**Comment by PVince81** [comment]
Yeah, so far I was very confused what the "State: -" meant.

**Comment by PVince81** [comment]
"Shared with: -" in latest ocis-phoenix not looking better :-D

**Comment by kulmann** [comment]
We did a little bit of visual prototyping in todays design initiative meeting. For that prototyping we added some wild guessed icons, to see how it looks like in a crowded state, i.e. when there are more indicators.

## Idea:
- the wording should express a state (like `Shared`, `Public`, `Tagged`, etc)
- label on the right of the indicator
- try smaller font size for indicator row
- hide labels on mobile

## Screenshots:

![Screenshot 2020-07-23 at 14 58 56](https://user-images.githubusercontent.com/3532843/88289625-d3c24700-ccf5-11ea-865c-bf69b3751ea8.png)
![Screenshot 2020-07-23 at 14 59 44](https://user-images.githubusercontent.com/3532843/88289632-d58c0a80-ccf5-11ea-8f27-9ad55707154c.png)
![Screenshot 2020-07-23 at 15 01 09](https://user-images.githubusercontent.com/3532843/88289635-d6bd3780-ccf5-11ea-9351-9e8ab9fff976.png)
![Screenshot 2020-07-23 at 15 01 26](https://user-images.githubusercontent.com/3532843/88289637-d755ce00-ccf5-11ea-93ef-fab462cf38ac.png)

Feedback appreciated. @micbar @LukasHirt

**Comment by LukasHirt** [comment]
I have quite mixed feelings about this. I like that the indicators have a label but seeing then multiple ones feels like too many items screaming for the attention.

**Comment by kulmann** [comment]
We could try with even smaller font and grey text...

**Comment by LukasHirt** [comment]
> We could try with even smaller font and grey text...

Sounds like worth the try 🙂

**Comment by pascalwengerter** [comment]
@tbsbdr could you update this to the current state of things eventually? :)

**Comment by tbsbdr** [comment]
### Current state of discussion
... and next steps:
- **purpose** the sharing indicators proofed to be quite helpful for the users question "who has access?" at first glance
- **effective** in the usability test the sharing indicators functionality was recognised correctly and gave safety to the users, as their (prior) sharing-action was reflected by  the indicators (btw: fulfills "" visibility of system status https://www.nngroup.com/articles/visibility-system-status/)
- **similar purpose** we have a similar component in the shared with others section - the "OcAvatarGroup" which shows even more detailed infos at first glace (number of shares and persons, link). I would love to see the "OcAvatarGroup" in the Files List, but this would slow down the system as it would eat lots of resources to collect all the sharing infos recursively -> practically not feasible

**Problems with the current solution**
- **conflict: condensed view** as we also need a more condensed view in the future, the 2-lined Filename+Sharing indicator make it difficult to lower the whole row-height
- **small clickable area** The indicators have only a clickable area of 24 x 24 px. ideally a clickable area is 44 px x 44 px (https://www.w3.org/WAI/WCAG21/Understanding/target-size.html)

### solution
long story short
--> move the sharing indicators into a separate column (as it was already done some time ago ;-) )
--> visually distinct the sharing indicators from the quickactions

compare:
![image](https://user-images.githubusercontent.com/26610733/119982255-e618ab80-bfbe-11eb-847f-215c96cdc677.png)

**Comment by pascalwengerter** [comment]
Superseded by more recent #5976
